discord/bot.go

```diff
@@ -355,7 +355,7 @@ func updatesListener(dg *discordgo.Session, guildID string, socketUpdates *chan
 
 						if updated {
 							//log.Println("Player update received caused an update in cached state")
-							if isAliveUpdated && guild.AmongUsData.GetPhase() == game.TASKS {
+							if isAliveUpdated && guild.AmongUsData.GetPhase() == game.TASKS && !guild.PersistentGuildData.UnmuteDeadDuringTasks {
```

Comment:
This new variable would actually only have an effect on the updating of the bot's status message, and not on immediately muting/unmuting the player right here. To accomplish this, I think you'd have to call `handleTrackedMembers`, which is found in guild.go. And for this case, with delay 0 and NoPriority